packages/jaeger-ui/src/components/App/Page.css

```diff
@@ -34,3 +34,7 @@ limitations under the License.
 .Page--content--no-embedded {
   top: var(--nav-height);
 }
+
+.ant-menu-item:hover {
+  background-color: transparent !important;
+}
```

Comment:
In antd 4x, the TopNav contains the hover effect by default, which is not what we had earlier.
This CSS property disables that behavior.

Reply:
don't understand this one. The About Jaeger menu also self-activates on hover.

Reply:
> The About Jaeger menu also self-activates on hover.

I just checked the live build on the previous version (3.x) without any change. It showed the same behavior there too.
If that was a bug, I would fix it in next PR.

About that CSS change, here's how it behaves without it:

[screen-capture.webm](https://github.com/jaegertracing/jaeger-ui/assets/94157520/1dd49920-195a-4433-8d68-122d71fd5990)

Reply:
lgtm